backend/src/plugins/fastify-roles/README.md

```diff
@@ -0,0 +1,17 @@
+# Fastify Roles plugin
```

Comment:
Nice work including a README for this plugin. It would make it easier for you to publish it on its own.